package.json

```diff
@@ -31,7 +31,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "build:extension": "webpack -p --module-bind js=babel-loader ./extension/extension.js ./extension/build/extension.js",
+    "build:extension": "webpack -p --module-bind js=babel-loader --devtool=source-map ./extension/extension.js ./extension/build/extension.js",
```

Comment:
I'll triple check - however everything that gets in the zip gets bundled in the extension, there's no explicit declaration file for assets.